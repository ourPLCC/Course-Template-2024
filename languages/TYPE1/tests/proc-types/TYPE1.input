declare f : [=>int]
declare y : int
define f = proc():int y
define y = 3
.f() 