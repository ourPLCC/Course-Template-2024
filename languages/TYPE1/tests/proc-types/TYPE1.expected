f:[=>int]
y:int
f:[=>int]
y:int
3